exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/CoreSchemaProxy.java

```diff
@@ -100,15 +100,15 @@ long getHeight() {
    *
    * @throws IllegalArgumentException if the height is negative or there is no block at given height
    */
-  ProofListIndexProxy<HashCode> getBlockTransactions(long height) {
-    checkArgument(height >= 0, "Height shouldn't be negative, but was %s", height);
-    long actualHeight = getHeight();
+  ProofListIndexProxy<HashCode> getBlockTransactions(long blockHeight) {
+    checkArgument(blockHeight >= 0, "Height shouldn't be negative, but was %s", blockHeight);
+    long blockchainHeight = getHeight();
```

Comment:
That's a good distinction (blockHeight vs blockchainHeight) that might help to avoid confusions in other places too.